### PR TITLE
Rename 'nirfmxbt' to 'nirfmxbluetooth'.

### PR DESCRIPTION
**Pull request by tdunkle**
### What does this Pull Request accomplish?

This pull request renames `nirfmxbt` to `nirfmxbluetooth` throughout grpc-device.

### Why should this Pull Request be merged?

We want to be more consistent with the nomenclature used in the NI-RFmx driver, plus spelling out 'bluetooth' makes grpc-device more easily searchable.

### What testing has been done?

grpc-device builds locally with this change, and all of the bluetooth tests pass:

image

### CODE REVIEW

File: generated/nirfmxbluetooth/nirfmxbluetooth.proto [hunk 1]
```diff
@@ -0,0 +1,2025 @@
+
+//---------------------------------------------------------------------
+// This file is automatically generated. All manual edits will be lost.
+//---------------------------------------------------------------------
+// Proto file for the NI-RFMXBT Metadata
+//---------------------------------------------------------------------
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "com.ni.grpc.nirfmxbt";
```
Comment (astarche) [inline generated/nirfmxbluetooth/nirfmxbluetooth.proto:10]: All of these package names should be updated too.

File: generated/nirfmxbluetooth/nirfmxbluetooth.proto [hunk 2]
```diff
@@ -0,0 +1,2025 @@
...
+option java_outer_classname = "NiRFmxBT";
```
Comment (astarche) [inline generated/nirfmxbluetooth/nirfmxbluetooth.proto:11]: What's our capitalized name? `Bluetooth`?

File: generated/nirfmxbluetooth/nirfmxbluetooth.proto [hunk 3]
```diff
@@ -0,0 +1,2025 @@
...
+  rpc TXPFetchLECTEReferencePeriodPowers(TXPFetchLECTEReferencePeriodPowersRequest) returns (TXPFetchLECTEReferencePeriodPowersResponse);
+  rpc TXPFetchLECTETransmitSlotPowers(TXPFetchLECTETransmitSlotPowersRequest) returns (TXPFetchLECTETransmitSlotPowersResponse);
+  rpc TXPFetchLECTETransmitSlotPowersArray(TXPFetchLECTETransmitSlotPowersArrayRequest) returns (TXPFetchLECTETransmitSlotPowersArrayResponse);
+  rpc TXPFetchPowerTrace(TXPFetchPowerTraceRequest) returns (TXPFetchPowerTraceResponse);
+  rpc TXPFetchPowers(TXPFetchPowersRequest) returns (TXPFetchPowersResponse);
+  rpc WaitForAcquisitionComplete(WaitForAcquisitionCompleteRequest) returns (WaitForAcquisitionCompleteResponse);
+  rpc WaitForMeasurementComplete(WaitForMeasurementCompleteRequest) returns (WaitForMeasurementCompleteResponse);
+}
+
+enum NiRFmxBTAttribute {
```
Comment (astarche) [inline generated/nirfmxbluetooth/nirfmxbluetooth.proto:149]: Should also be bluetooth.

**Review by SparkingSpork — APPROVED** [review]

File: generated/nirfmxbluetooth/nirfmxbluetooth.proto [hunk 4]
```diff
@@ -0,0 +1,2025 @@
...
+  TXP_AVERAGING_ENABLED_TRUE = 1;
+}
+
+enum TxpBurstSynchronizationType {
+  TXP_BURST_SYNCHRONIZATION_TYPE_NONE = 0;
+  TXP_BURST_SYNCHRONIZATION_TYPE_PREAMBLE = 1;
+  TXP_BURST_SYNCHRONIZATION_TYPE_SYNC_WORD = 2;
+}
+
+enum NiRFmxBTInt32AttributeValues {
```
Comment (astarche) [inline generated/nirfmxbluetooth/nirfmxbluetooth.proto:419]: Another BT.

**Review by astarche — APPROVED** [review]

File: generated/nirfmxbluetooth/nirfmxbluetooth.proto [hunk 5]
```diff
@@ -0,0 +1,2025 @@
...
+option java_package = "com.ni.grpc.nirfmxbt";
+option java_outer_classname = "NiRFmxBT";
+option csharp_namespace = "NationalInstruments.Grpc.NiRFmxBT";
+
+package nirfmxbluetooth_grpc;
+
+import "nidevice.proto";
+import "session.proto";
+
+service NiRFmxBT {
```
Comment (mgumble) [inline generated/nirfmxbluetooth/nirfmxbluetooth.proto:19]: Here is another NiRFmxBT that was generated

File: generated/nirfmxbluetooth/nirfmxbluetooth.proto [hunk 6]
```diff
@@ -0,0 +1,2025 @@
...
+  NIRFMXBT_INT32_TWENTY_DB_BANDWIDTH_AVERAGING_ENABLED_FALSE = 0;
+  NIRFMXBT_INT32_TWENTY_DB_BANDWIDTH_AVERAGING_ENABLED_TRUE = 1;
+  NIRFMXBT_INT32_TXP_AVERAGING_ENABLED_FALSE = 0;
+  NIRFMXBT_INT32_TXP_AVERAGING_ENABLED_TRUE = 1;
+  NIRFMXBT_INT32_TXP_BURST_SYNCHRONIZATION_TYPE_NONE = 0;
+  NIRFMXBT_INT32_TXP_BURST_SYNCHRONIZATION_TYPE_PREAMBLE = 1;
+  NIRFMXBT_INT32_TXP_BURST_SYNCHRONIZATION_TYPE_SYNC_WORD = 2;
+}
+
+enum NiRFmxBTStringAttributeValuesMapped {
```
Comment (reckenro) [inline generated/nirfmxbluetooth/nirfmxbluetooth.proto:494]: Found another BT.

File: source/codegen/metadata/nirfmxbluetooth/config.py [hunk 7]
```diff
@@ -6,7 +6,7 @@
     'service_class_prefix': 'NiRFmxBT',
     'java_package': 'com.ni.grpc.nirfmxbt',
     'csharp_namespace': 'NationalInstruments.Grpc.NiRFmxBT',
-    'namespace_component': 'nirfmxbt',
+    'namespace_component': 'nirfmxbluetooth',
```
Comment (reckenro) [inline source/codegen/metadata/nirfmxbluetooth/config.py:9]: I think you'll also need to update 'service_class_prefix', 'driver_name', 'java_package', 'csharp_namespace'.

**Review by reckenro — APPROVED** [review]

File: generated/nirfmxbluetooth/nirfmxbluetooth.proto (hunk 2 above)
Comment (tdunkle) [reply generated/nirfmxbluetooth/nirfmxbluetooth.proto]: Yes, and these package names are fixed now.

**Review by mgumble — APPROVED** [review]

**Pull request merged**